Explorer/Assets/DCL/UI/ConnectionStatusPanel/StatusEntry/IStatusEntry.cs

```diff
@@ -0,0 +1,48 @@
+using ECS.SceneLifeCycle.CurrentScene;
+using System;
+using System.Collections.Generic;
+
+namespace DCL.UI.ConnectionStatusPanel.StatusEntry
+{
+    public interface IStatusEntry
+    {
+        enum Status
+        {
+            Poor,
+            Good,
+            Excellent,
+        }
+
+        public void ShowReloadButton(Action onClick);
+
+        public void ShowStatus(string status);
+
+        public void HideStatus();
+    }
+
+    public static class StatusEntryExtensions
```

Comment:
Why extension methods for this cause, and not directly implement it in the view?

Reply:
It relies only on interface exposed properties, it doesn't make a sense to copy it to each implementation

Reply:
You have only one implementation of this interface, thus we dont know if we need this on the future implementations (if any at all exists)

Reply:
It's not responsibility of the StatusEntry to handle different enums, it relies only on `string` message and shows it. Bringing those static dependency-free implementation to StatusEntry will just bloat it with no benefits. Being a part of interface it could be united with `IStatusEntry` itself, I prefer extensions especially they are both in the same file and could be easily navigated

Reply:
On this case you are hardcoding text that will be shown to the user. Whenever we have to implement localization features, it will be better to access the localization component directly from the view since it can be serialized through the editor, so i think it is better that the view receives the enum instead of the text. But anyway that is my opinion, i will approve nonetheless

Reply:
Good point! We'll do it when we handle localization

Reply:
Even if we are not to localize it straight-away it could be the responsibility of the view to project enums onto the displayed values. As for me it feels unnecessary to detach it to extra extensions because it won't be reused by any other implementation anyway.

It also could be integrated into `ConnectionStatusPanelController` as it's a view controller and has a responsibility to provide values for the view including localisation in more complex scenarios.